ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -31,7 +43,7 @@ const buildAuthorSnapshot = (roles = false) => {
     };
 
     if (roles) {
-        return {...authorSnapshot, roles: Array(1).fill(roleSnapshot)};
+        return { ...authorSnapshot, roles: Array(1).fill(roleSnapshot) };
```

Comment:
No need for extra spaces here 🙃

Reply:
I apologize, my bad. I've resolved this, thank you.